src/datasource.ts

```diff
@@ -275,7 +289,30 @@ export default class CogniteDatasource {
           aggregation: q.aggregates,
         }
       }));
-    }, []);
+
+      // assign labels to each timeseries
+      if (target.tab === Tab.Timeseries) {
+        if (target.label.indexOf("{{") < target.label.lastIndexOf("}}")) {
```

Comment:
This is beautiful, but it looks to me like it will match with the label expression `labels are cool}}`, as (`indexOf("{{")` would be -1 which is < whatever index `}}` is on.

Reply:
Good point, I'll change it to do a simple regex match on /{{.*}}/